8_class_components/src/components/Navbar.js

```diff
@@ -1,11 +1,23 @@
 import logo from "../assets/logo.jpeg";
 
+import { Link } from "react-router-dom";
+
 const Navbar = () => {
 	return (
 		<div className="navbar">
 			<div className="logo">
 				<img src={logo} alt="logo"></img>
-				<h1 className="title">Web Pirates</h1>
+				<h1 className="title link">Web Pirates</h1>
+			</div>
+
+			<div className="nav-links">
+				<Link to="/" className="link">
+					Home
```

Comment:
can you use prettier for formatting?

Reply:
Yes, prettier auto-formatted it this way